spec/models/student_registration_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+describe StudentRegistration do
+  context 'when registering' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.